app/src/views/OperationalLearnings/styles.module.css

```diff
@@ -0,0 +1,9 @@
+.title{
+    display:flex;
+    flex-direction: row;
+    justify-content:space-between;
+    margin-top: 5rem;
+}
+.components-summaries{
+    margin-top: 5rem;
+}
```

Comment:
you seem to have referenced `styles.operationalLearnings` in the code. However, it is not defined anywhere. We should only use those styles that are defined and also remove unused styles from the `css` file. This is something we should be mindful about.